### ISSUE TITLE
Country flags not showing for phone field in submissions

### ISSUE THREAD

**Issue by JeanLucEsser** [verbb/formie#982]
### Describe the bug

In the CP, when looking at submissions, the country flags do not show.

<img width="125" alt="image" src="https://user-images.githubusercontent.com/14908712/178285051-d1ee82c2-6757-4afd-ba59-42e52bb893a0.png">

### Steps to reproduce

1. Create a form with a phone number field (with country codes)
2. Fill in the form and go see a submission

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.1.3

### Plugin version

2.0.0

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by JeanLucEsser** [comment]
(oops sorry thought it would help if I closed my resolved issues - won't do it again) ;)

**Comment by engram-design** [comment]
All good! I appreciate a closed issue as much as the next person 😉

**Comment by engram-design** [comment]
Fixed in [2.0.1](https://github.com/verbb/formie/releases/tag/2.0.1)
